src/Backend/Modules/Blog/Engine/Model.php

```diff
@@ -1113,67 +1101,50 @@ public static function updateCategory(array $item, array $meta = null): int
         return $updated;
     }
 
-    /**
-     * Update an existing comment
-     *
-     * @param array $item The new data.
-     *
-     * @return int
-     */
-    public static function updateComment(array $item): int
+    public static function updateComment(array $item)
```

Comment:
:void

Reply:
fixed in 18adc30c0965878417a31b49881be34846e4911a